nautobot/dcim/forms.py

```diff
@@ -2928,43 +2963,68 @@ class PowerPortCreateForm(ModularComponentCreateForm):
         "type",
         "maximum_draw",
         "allocated_draw",
+        "power_factor",
         "description",
         "tags",
     )
 
 
 class PowerPortBulkCreateForm(
-    form_from_model(PowerPort, ["type", "maximum_draw", "allocated_draw", "tags"]),
+    form_from_model(PowerPort, ["type", "maximum_draw", "allocated_draw", "power_factor", "tags"]),
     DeviceBulkAddComponentForm,
 ):
+    # Override the auto-generated power_factor field to make it properly optional
+    power_factor = forms.DecimalField(
+        max_digits=4,
+        decimal_places=2,
+        min_value=0.01,
+        max_value=1.00,
+        required=False,
+        initial=0.95,
+        help_text="Power factor (0.01-1.00) for converting between watts and VA.",
+    )
+
     field_order = (
         "name_pattern",
         "label_pattern",
         "type",
         "maximum_draw",
         "allocated_draw",
+        "power_factor",
         "description",
         "tags",
     )
 
 
 class ModulePowerPortBulkCreateForm(
-    form_from_model(PowerPort, ["type", "maximum_draw", "allocated_draw", "tags"]),
+    form_from_model(PowerPort, ["type", "maximum_draw", "allocated_draw", "power_factor", "tags"]),
     ModuleBulkAddComponentForm,
 ):
+    # Override the auto-generated power_factor field to make it properly optional
```

Comment:
That's odd - `form_from_model()` is already supposed to mark all of the form fields it creates as `required=False`